### PR TITLE
Adding the Netherlands national page

### PR DESCRIPTION
**Pull request by bedroesb**

### CODE REVIEW

**Comment by bedroesb** [comment]
@lauportell @floradanna Can you discuss how we handle linking towards multiple places in the https://www.health-ri.nl/?

**Comment by bedroesb** [comment]
@MijkeJetten Don't forget to add the correct `page_id`s to the resources :)

File: pages/national_resources/nl_resources.md [hunk 1]
```diff
@@ -3,38 +3,221 @@
...
+contributors: [Mijke Jetten, Celia van Gelder, Rob Hooft, Mascha Jansen]
+coordinators: [Mijke Jetten]
 
 training:
   - name: Training in TeSS
     registry: TeSS
-    url: <!--- https://tess.elixir-europe.org/materials?node=NODENAME --->
+    registry_url: https://tess.elixir-europe.org
+    url: https://tess.elixir-europe.org/nodes/netherlands
   - name: ELIXIR NODENAME community in Zenodo
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply pages/national_resources/nl_resources.md:14]: I am not following :) the [current link ](https://tess.elixir-europe.org/nodes/netherlands)is actually a better one than the one in the template in my opinion.

**Comment by lauportell** [comment]
> > @lauportell @floradanna Can you discuss how we handle linking towards multiple places in the https://www.health-ri.nl/?
> 
> I would suggest to only link once to the https://www.health-ri.nl website (instead of having multiple link to subpages) and shortly summarise what info can be found on the website in the description.

I think it make sense to have the main link in the tools table and have a description of the different parts you can find in the text.

**Comment by floradanna** [comment]
@MijkeJetten  I noticed that at leat 10 links added as separate National resources are actually sub-pages of the main website https://www.health-ri.nl/services/ . Do you think you can just add the main link only once?

**Comment by lauportell** [comment]
Hi @MijkeJetten, did you hav time to have a look at the links you have added? 10 links added as separate National resources are actually sub-pages of the main website https://www.health-ri.nl/services/ . Do you think you can just add the main link only once?

**Comment by MijkeJetten** [comment]
Hi @lauportell @floradanna, I think I'm done updating - managed to solve a lot of "double" references to Health-RI and some other stuff as well. Already contacted @bedroesb  if it's good enough to do another pull request, we will sit together Monday (I'm really a Github noob, sry).

**Review by lauportell — APPROVED** [review]

File: pages/national_resources/nl_resources.md [hunk 2]
```diff
@@ -1,40 +1,136 @@
...
+    description: Biobanking Netherlands makes biosamples, images and data findable, accessible and usable for health research.
+    how_to_access: 
+    instance_of: 
+    related_pages:
+      tool_assembly: 
+      your_domain:
+      your_role: [researcher]
+      your_tasks:
+    url: https://www.bbmri.nl
+  - name: Jupyter Notebooks
```
Comment (bedroesb) [inline pages/national_resources/nl_resources.md:43]: This is a generic tool that is already part of the main tools table

File: pages/national_resources/nl_resources.md [hunk 3]
```diff
@@ -1,40 +1,136 @@
...
+      your_role: [researcher]
+      your_tasks:
+    url: https://www.dtls.nl/technology-hotels/list
+  - name: Dutch COVID-19 Data Support Programme
+    description: To support investigators and health care professionals with tools and services in their search for ways to overcome the pandemic and its health consequences.
+    how_to_access: 
+    instance_of: 
+    related_pages:
+      tool_assembly: 
+      your_domain:
```
Comment (bedroesb) [inline pages/national_resources/nl_resources.md:79]: can we tag this resource with human data or is that not correct?

**Pull request merged**